api/routes/transactions.js

```diff
@@ -3,23 +3,34 @@
  */
 
 const express = require('express');
-const Sequelize = require('sequelize');
 const router = express.Router();
 
+const logger = require('../logger');
 const Transaction = require('../models/transaction');
 
 router.get('/', async (req, res, next) => {
-    const transactions = await Transaction.findAll();
-    res.send(transactions);
+    try {
+        const transactions = await Transaction.findAll();
+        res.json(transactions);
+    } catch(err) {
+        res.status(500).send(err.message);
+        logger.error('Error transaction get', err);
```

Comment:
please change to human readable sentence: "An error occurred when getting transactions"